efficientdet/keras/train_lib.py

```diff
@@ -467,7 +473,12 @@ def test_step(self, data):
     """
     images, labels = data
     cls_outputs, box_outputs = self(images, training=False)
-    det_loss, _, _, _ = self._detection_loss(cls_outputs, box_outputs, labels)
+    det_loss, cls_loss, box_loss, box_iou_loss =\
```

Comment:
Similar change (better to avoid \)

Reply:
> Is it a bug?
> https://github.com/google/automl/blob/88b9a478cee2c4d79738af79d443475b79c19d45/efficientdet/iou_utils.py#L177
> 
> 
> pred_boxes should be [ymin, xmin, ymax, xmax] instead of box regression which is between [0,1].]]

Hmm, I haven't looked closely to iou_loss_utils.py. but it seems like " mask - tf.squeeze(_iou_per_anchor(pred_boxes, target_boxes, iou_type)))" should be " mask * (1 - tf.squeeze(_iou_per_anchor(pred_boxes, target_boxes, iou_type))))"

> And comments
> https://github.com/google/automl/blob/de4bdd67211fd36f54c7b19ae5a2a87618ad48e2/efficientdet/dataloader.py#L271
> 
> seems wrong.

Not sure about this.